src/event.js

```diff
@@ -357,6 +409,16 @@ PubfoodEvent.prototype.ANNOTATION_TYPE = {
 };
 
 /**
+ * @description Event annotation types<br>
+ * @type {object}
+  * @property {number} ONCE <b><code>1</code></b> : observer will execute at most once
+  * @property {number} ALL  <b><code>-1</code></b> : observer will execute for each of the emitted events for specified event name
+ */
+PubfoodEvent.prototype.OBSERVE_TYPE = {
```

Comment:
Ordinarily properties such as this should be static. However, to be consistent with the implementation this will be ok. `PubfoodEvent` is an internal singleton that _should_ not be instantiated outside of the library.